### PR TITLE
Align with rust-lang/rust/#58992

### PR DESCRIPTION
**Pull request by ljedrz**
Some adjustments needed after the most recent round of HirIdification.

### CODE REVIEW

File: clippy_lints/src/lib.rs [hunk 1]
```diff
@@ -272,7 +272,7 @@ pub mod zero_div_zero;
 pub use crate::utils::conf::Conf;
 
 mod reexport {
-    crate use syntax::ast::{Name, NodeId};
+    crate use syntax::ast::Name;
```
Comment (ljedrz) [inline clippy_lints/src/lib.rs]: aww yiss

**Comment by ljedrz** [comment]
As usual, you can branch the PR off so I can push it alongside the rust-lang PR,

**Comment by flip1995** [comment]
Done

**Comment by ljedrz** [comment]
The rust-lang PR was merged.

**Pull request merged**